active/social-media.md

```diff
@@ -0,0 +1,72 @@
+# Social Media Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official social media profiles on behalf of the Django Software Foundation.
+
+The goals of the group are:
+- Promote the use of Django among the World Wide Web development community.
+- Protect the framework's long-term viability by generating revenue through sponsor relationships/campaigns.
+
+The group opts to manage platforms that best achieve these goals - which may be a subset of where Django is or could be engaging.
+
+Delegated responsibilities:
+- The Chair can grant access to Django's official social media profiles and each member will have access.
+- Members can post content to Django's official social media profiles, in alignment with the working group's agreed goals, without a board review.
+- Members can like/repost/comment as Django, when it aligns with the working group's agreed goals, without a board review.
+- The working group can post content on djangoproject.com, when it aligns with the working group's agreed goals, without a board review.
```

Comment:
I wonder if there are established guidelines for this one already? :thinking:

Reply:
“on djangoproject.com” is vague - does this mean the blog, or somewhere else on the site?

Also, the blog is a different kind of channel as posts have a specific individual author, rather than “Django”.

Reply:
I have decided to remove it, I can't immediately see a need for doing a post here, so might not be required.

Reply:
I don’t feel strongly about this but would recommend adding this back, focused on the blog. From experience, when doing multi-channel promotions it’s really useful if all the channels can point to one shared resource (so a blog post in this context). I also suspect Django blog posts also have a lot of visibility in "social" channels due to the RSS feed (for example Discord).